common/src/main/java/org/corfudb/common/metrics/micrometer/MeterRegistryProvider.java

```diff
@@ -26,6 +31,7 @@
 public class MeterRegistryProvider {
     @Getter
     private static CompositeMeterRegistry meterRegistry;
+    private static final String PROVIDED_REGISTRY_DATA_FILE_PATH = "/registry.dat";
```

Comment:
Should this be configurable and passed via parameters?

Reply:
This exact file is a part of the protocol to provide metadata for custom registries. I will write a document on the exact implementation, so that it's more clear but the short answer is no, it should be static.